serde/src/java/org/apache/hadoop/hive/serde2/objectinspector/ObjectInspectorUtils.java

```diff
@@ -1654,27 +1652,4 @@ public static boolean hasAllFieldsSettable(ObjectInspector oi,
   private ObjectInspectorUtils() {
     // prevent instantiation
   }
-
-  /**
-   * Returns slot value used for ordering the fields to make it deterministic
-   * @param field : field of a given class
-   * @return
-   */
-  private static int getSlotValue(Field field) {
```

Comment:
Why are you removing this, you have that ReflectionUtils or so, can't we use that

Reply:
From JDK12 , Field class is field filtered out so we cannot access them via reflection. Added details in the ticket. 
https://issues.apache.org/jira/browse/HIVE-28643